src/main/java/com/minecolonies/coremod/commands/ActionMenu.java

```diff
@@ -0,0 +1,57 @@
+package com.minecolonies.coremod.commands;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.jetbrains.annotations.NotNull;
+
+import com.minecolonies.coremod.colony.permissions.ForgePermissionNodes;
+
+public class ActionMenu extends AbstractMenu
+{
+    @NotNull private final String description;
+    @NotNull private final ForgePermissionNodes forgePermissionNode;
+    @NotNull private final List<ActionArgument> actionArgumentList;
+    @NotNull private final Class<? extends IActionCommand> actionCommandClass;
+
+    public ActionMenu(
+            @NotNull final String description,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "description". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1172)